docs/cloudfoundry/cloudfoundry.yml

```diff
@@ -13,6 +13,12 @@ google_zone = "{{ZONE}}"
 network = "cf"
 public_subnetwork = "cf-public-#{google_region}"
 private_subnetwork = "cf-private-#{google_region}"
+
+# Google service account settings
+project_id = "{{PROJECT_ID}}"
+org, project = project_id.split(":")
+project_domain_prefix = [project, org].compact.join(".")
+cf_service_account = "cf-component@#{project_domain_prefix}.iam.gserviceaccount.com"
```

Comment:
just scratching my head at the console output, i'm pretty certain it's accurate though. this somewhat obfuscated code let's us handle both org/non org projects